elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQuery.java

```diff
@@ -30,14 +28,15 @@
 
 /**
  * Model for Async Query.
+ * ExecuteQueryHook is binded manually during the elide startup,
+ * after asyncexecutorservice is initialized.
  */
 @Entity
 @Include(type = "asyncQuery", rootLevel = true)
 @ReadPermission(expression = "Principal is Owner")
 @UpdatePermission(expression = "Prefab.Role.None")
 @DeletePermission(expression = "Prefab.Role.None")
 @LifeCycleHookBinding(hook = UpdatePrincipalNameHook.class, operation = CREATE, phase = PRESECURITY)
-@LifeCycleHookBinding(hook = ExecuteQueryHook.class, operation = CREATE, phase = POSTCOMMIT)
```

Comment:
Might as well make this consistent - let's remove all life cycle hook annotations from this class.